index.html

```diff
@@ -1,66 +1,354 @@
+                <span id="buttonText">See Project</span>
+              </button>
+            </div>
+          </div>
+          <div class="flex-container" id="cardWork">
+            <ul id="cardElements">
+              <li>
+                <h1 id="cardTitle">Profesional Art Printing Data</h1>
+              </li>
+              <li>
+                <h1 id="cardDesc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h1>
+              </li>
+              <li id="cardTags">
+                <ul class="flex-container" id="tagGroup">
+                  <li class="tag">
+                    <a class="tagText" id="html">html</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="bootstrap">bootstrap</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="Ruby">Ruby</a>
+                  </li>
+                </ul>
+            </ul>
+            <div id="cardButton">
+              <button class="button" type="button">
+                <span id="buttonText">See Project</span>
+              </button>
+            </div>
+          </div>
+          <div class="flex-container" id="cardWork">
+            <ul id="cardElements">
+              <li>
+                <h1 id="cardTitle">Profesional Art Printing Data</h1>
+              </li>
+              <li>
+                <h1 id="cardDesc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h1>
+              </li>
+              <li id="cardTags">
+                <ul class="flex-container" id="tagGroup">
+                  <li class="tag">
+                    <a class="tagText" id="html">html</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="bootstrap">bootstrap</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="Ruby">Ruby</a>
+                  </li>
+                </ul>
+            </ul>
+            <div id="cardButton">
+              <button class="button" type="button">
+                <span id="buttonText">See Project</span>
+              </button>
+            </div>
+          </div>
+      </div>
+      <div class="flex-container" id="aboutMyself">
+        <ul class="flex-container" id="aboutMe">
           <li>
-            <h1 id="primary-text">Hey There. I'm Eduardo</h1>
+            <h1 id="amTitle">About me</h1>
           </li>
           <li>
-            <h1 id="supporting-text">I'am a Software Developer</h1>
+            <h1 id="amDesc">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</h1>
           </li>
```

Comment:
The paragraph ("I'm a software ...") should be a `p` tag, This way will be semantically correct and the screen readers and other technologies will thank you.